pkg/labels/matcher.go

```diff
@@ -86,3 +89,46 @@ func (m *Matcher) Matches(s string) bool {
 	}
 	panic("labels.Matcher.Matches: invalid match type")
 }
+
+type Matchers []*Matcher
+
+func (ms Matchers) Len() int      { return len(ms) }
+func (ms Matchers) Swap(i, j int) { ms[i], ms[j] = ms[j], ms[i] }
+
+func (ms Matchers) Less(i, j int) bool {
+	if ms[i].Name > ms[j].Name {
+		return false
+	}
+	if ms[i].Name < ms[j].Name {
+		return true
+	}
+	if ms[i].Value > ms[j].Value {
+		return false
+	}
+	return ms[i].Value < ms[j].Value
```

Comment:
```suggestion
	return ms[i].Value < ms[j].Value
```

Also, does it make sense to sort by `.Type` if both `.Name` and `.Value` are equal?

Reply:
I don't see any use case for it...

Reply:
But it would be legal. To be conservative, I'd actually prefer if we included Type as the ultimate sorting criteria. Imagine someone accidentally puts in `{foo="bar", foo!="bar"}`. At least the rendered config should always use the same order to help that poor fellow to find their mistake.

Reply:
Done